gobblin-runtime/src/main/java/org/apache/gobblin/runtime/fork/Fork.java

```diff
@@ -24,6 +24,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.github.rholder.retry.RetryException;
```

Comment:
Unused import.

Reply:
That should have been caught by checkstyle plugin, but apparently it is configured to check only a very small subset of things. I'll send separate PRs to enable more checks.